src/api.cc

```diff
@@ -333,8 +347,25 @@ CheckInResult AkliteClient::CheckInLocal(const LocalUpdateSource* local_update_s
   LOG_INFO << "Updating the local TUF repo with metadata located in " << local_update_source->tuf_repo << "...";
   try {
     tuf_repo_->updateMeta(repo_src);
+    LOG_INFO << "The local TUF repo has been successfully updated";
+  } catch (const Uptane::SecurityException& exc) {
+    const std::string err{exc.what()};
+    LOG_ERROR << "Failed to update the local TUF repo SecurityException: " << err;
+    return CheckInResult{CheckInResult::Status::SecurityError, "", {}};
+  } catch (const Uptane::ExpiredMetadata& exc) {
+    const std::string err{exc.what()};
+    LOG_ERROR << "Failed to update the local TUF repo ExpiredMetadata: " << err;
```

Comment:
`LOG_ERROR << "Failed to update the local TUF repo; TUF meta are expired: " << err;`

Reply:
Done.